src/ProjectOrigin.Wallet.Server/Program.cs

```diff
@@ -10,8 +12,12 @@
 // Add services to the container.
 builder.Services.AddGrpc();
 
+builder.Services.AddSingleton<IDapperContext, DapperContext>();
```

Comment:
I think we need to have a look at this.

A Context object is for a specific connection in my world, I would probably look at this like a factory, so it should maybe be a IDapperConnectionFactory or IDBConnectionFactory?

Reply:
You are right, it is a factory. I prefer calling it `IDbConnectionFactory` mainly due to the fact, that it does not have anything to do with Dapper.